suites/pacific/cephadm/tier-1_cephadm-clients.yaml

```diff
@@ -0,0 +1,173 @@
+#       - skip-monitoring-stack: true
+#       - orphan-initial-daemons: true
+#       - registry-json: <registry-URL>
+#       - fsid: <cluster-fsid>
+#       - mon-ip: <monitor IP address: Required>
+#       - config: <ceph config options to be set during bootstrap>
+#   (2) Copy SSH keys to nodes.
+#   (3) Add nodes to cluster with address and role labels attached to it using Host spec yaml file.
+#   (4) Deploy services using apply spec option, (" ceph orch apply -i <spec_file>)
+#       - 3 Mon on node1, node2, node3 using host placements.
+#       - MGR using placement using label(mgr).
+#       - addition of OSD's using "all-avialable-devices" option.
+#       - alertmanager on node1, node2 using label "alert-manager".
+#       - grafana and prometheus on node1 using host placement with limit.
+#       - crash and node-exporter on all nodes using placement="*".
+#   (5) Configure client
+#   (6) Using cephadm-clients playbook distribute keyring with custom destination name
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      config:
+         is_production: True
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap
+      desc: cephadm cluster bootstrap
+      module: test_bootstrap.py
+      polarion-id: CEPH-83573720
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          skip-monitoring-stack: true
+          orphan-initial-daemons: true
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Host addition with spec file
+      desc: add hosts using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574726
+      config:
+        steps:
+        - config:
+            service: host
+            command: set_address
+            args:
+              node: node1
+        - config:
+            service: host
+            command: label_add
+            args:
+              node: node1
+              labels: apply-all-labels
+        - config:
+            command: apply_spec
+            service: orch
+            specs:
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node2
+                 - node3
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node4
+      abort-on-fail: true
+  - test:
+      name: Service deployment with spec
+      desc: Add services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83573746,CEPH-83574727
```

Comment:
We are trying to avoid multiple test case ids as the framework cannot pinpoint which suite has failed. Hence, could we revisit this one?